documentation/configuration.markdown

```diff
@@ -35,6 +35,8 @@ The default Logback configuration file is at `/etc/puppetserver/logback.xml` or
 
 Puppet Server relies on `logrotate` to manage the log file, and installs a configuration file at `/etc/logrotate.d/puppetserver` or `/etc/logrotate.d/pe-puppetserver`.
 
+For some tips on advanced logging configuration, including information about configuring your system to send log data to logstash, see the [Advanced Logging Configuration](./config_logging_advanced.html) documentation.
```

Comment:
The logging doc doesn't really cover sending log data to logstash, it just refers to third-party tools that do it. And theoretically, third-party tools could send log data to anything, not just logstash.

Calling it "advanced logging configuration" implies there's a basic way to configure Puppet Server logging. We should probably describe (either in this section, in config_logging_advanced, or a separate doc) how to make basic changes, like switching logging modes from info to debug. I've got a branch [on my repo](https://github.com/gguillotte/puppet-server/blob/logback-docs/documentation/config_file_logbackxml.md) to try to tackle that.